config/system.php

```diff
@@ -91,5 +91,6 @@
     'php_max_execution_time' => '-1',
     'ajax_timeout' => '600000',
     'pcre_backtrack_limit' => '-1',
+    'glide_prepend_siteroot' => true,
```

Comment:
Please move this into the `config/assets.php` file.
I think a good place would to name it `prepend_site_root` inside the `image_manipulation` array. Right after `route`. Here: https://github.com/statamic/cms/blob/master/config/assets.php#L18